src/modules/esl-toggleable/core/esl-toggleable.ts

```diff
@@ -314,8 +324,12 @@ export class ESLToggleable extends ESLBaseElement {
 
   /** Actions to execute on show request */
   @bind
-  protected _onShowRequest(): void {
-    this.show();
+  protected _onShowRequest(e: CustomEvent<ESLShowRequestDetails>): void {
+    const detail = e.detail;
+    this.show(!isMatches(this, detail.ignore) ?
```

Comment:
Please recheck the following AC of the story:
 - If the current element doesn't matches passed selector or predicate of `ignore` key then element *skips* `show` processing
 - it is recommended to pass `showDelay` with the `delay` key value if it is numeric
 - `parmas` key can override show params if passed (with an upper priority)
 - if none of `params` and `delay` passed then `show` requesed with an empty `{} or undefined` params